### ISSUE TITLE
[Discharge Upgrade Wizard] Results screen information is missing for many Air Force and some other military branch scenarios

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#19785]
## Description

@chriskim2311 Note that Air Force isn't the only branch missing info. See this [ticket ](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19599) that will be closed as a dupe for further info (that will be moved here)

The results sceen is missing most of it's information for particular path(s) for Air Force results. [Example scenarios here](https://docs.google.com/spreadsheets/d/1slM0u2vTfA6cAFluwWNZ3yEdA_hpdOaq8g7MCksnqLY/edit?gid=1357877297#gid=1357877297).

**Examples of information missing are below, based on the MS Word document provided by CAIA:**

- **If Question 3 = 1 (PTSD)** SHOW: 
  - Because you answered that your discharge was related to posttraumatic stress disorder (PTSD) or other mental health conditions, the DoD will apply “liberal consideration” to your case. In 2014, the DoD recognized that many Veterans had received discharges due to behavior connected to their previously undiagnosed or undocumented PTSD or other mental health condition.  

- **If Question 3 = 1 or 2 AND IF Question 2= 2001-2009 only, SHOW:**
Step 1 (Optional): Apply to the Physical Disability Board of Review

If you believe your disability rating for TBI, PTSD, or other mental health conditions is too low, consider applying to the Physical Disability Board of Review (PDBR). The DoD created the PDBR specifically to review appeals about low disability ratings for Veterans discharged between 2001 and 2009. Some Veterans discharged during this period of time received lower disability ratings than they deserved, especially if they suffered from TBI, PTSD, or other mental health conditions. If you were discharged during this period of time and previously received a disability rating of 20% or lower, you're eligible to apply to the PDBR for review. The PDBR does **not** issue discharge upgrades and cannot review conditions not listed in your military record before your separation. But, if the PDBR finds that your disability rating was unjustly low, it may help you make your case to upgrade your discharge.

[Learn more about PDBR reviews](https://health.mil/PDBR).

[Apply for a PDBR review](https://health.mil/Military-Health-Topics/Conditions-and-Treatments/Physical-Disability/Disability-Evaluation/Physical-Disability-Board-of-Review/PDBR-Application-Process).

Step 1: Download and fill out DoD Form
**Important tips for completing DOD Form 149**
- Pay special attention to item 14, which asks for the reason for your change. Most Veterans attach additional pages to answer this question. 
**If 3==1,2,4 SHOW:**
For discharges related to PTSD or other mental health conditions, be sure to answer these questions to make the strongest case:
- Did you have a condition that may explain or contribute to the reason you were discharged?
- Did that condition start or get worse during your military service?
- Why does the condition directly explain or contribute to the discharge?
- Why does the condition carry more weight than any other reasons you may have been discharged for?

**Note:** You're generally only eligible for reconsideration if you have new evidence to present that wasn't available when you applied last time. Make sure you're clear about exactly what that new evidence is. Additionally, changes in DoD policy, like the new consideration guidelines for PTSD, TBI, and sexual assault or harassment, can qualify you for reconsideration. 
 
**IF BCMR/BCNR SHOW:**
- Item 16 asks for the date when you discovered the error or injustice you‘re asking the Board to address. If it‘s been more than 3 years since you found this error or injustice, you‘ll need to include a reason why the Board should consider your application. Examples of good reasons include new evidence you‘ve found to support your claim, or recent changes in policy (like liberal consideration for PTSD, TBI, or military sexual assault or harassment). These kinds of reasons will make it more likely for the Board to decide in your favor. The 3-year time limit isn‘t a strict rule, so don‘t let it keep you from applying if you think you have a strong case.
- Item 17 asks if you're willing to appear in person before the Board in Washington, DC. The Board rarely asks Veterans to appear in person, but if you say you’re willing to do so, it may help show how serious you are about your case.

Step 2 and 3 Info TBD:

## Screenshots

This is a screenshot of the abbreviated results screen showing missing information.

![Image](https://github.com/user-attachments/assets/78448d8d-748f-444b-ab0f-8862b44b8c91)

### User story

**AS A** Veteran of the Air Force who is using Discharge Upgrade Wizard
**I WANT** to have all the information I need on the results screen
**SO THAT** I know what my next steps are and aren't frustrated by my experience or with the VA

## Testing & QA

- [QA Plan](https://docs.google.com/spreadsheets/d/1slM0u2vTfA6cAFluwWNZ3yEdA_hpdOaq8g7MCksnqLY/edit?pli=1&gid=1357877297#gid=1357877297) specifically for this has been started, but not yet completed.

### Scope / Impact analysis
_What, if anything, could break as a result of this change?_
_Engineer should assess this when approaching PR._

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
- [ ] **Functional testing:** @FranECross 
- [ ] **Code review:** 
- [ ] **Design review:** N/A
- [ ] **Accessibility review:** @laflannery 
- [ ] **Product acceptance:** @FranECross 

## AC / Expected behavior
- [ ] the results screen for the Air Force follows all the logic presented and detailed in the [MS Word document](https://dvagov.sharepoint.com/:w:/s/SitewidePublicWebsites/EZ63meOuaOJBgq4EL1p9pvQB6TUrm_C5Asl9Es1fsgKGOQ?email=Laura.Flannery%40va.gov&e=c8OVtr)

**Comment by chriskim2311** [comment]
@FranECross I was doing some digging into the logic behind this link showing by itself without supporting information. I found the original ticket and the ask for what we see on prod today. Can you take a look and verify that we want to implement these changes still? Maybe ask Joshua Gibbs/CAIA about it as well. https://github.com/department-of-veterans-affairs/va.gov-team/issues/23071

**Comment by FranECross** [comment]
@chriskim2311 Remember the most recent guidance from Joshua was to provide the PDF and the link to the website (referencing the recent tickets you worked on for him, and the UX work that Dave or Jordan did. So we'll want to emulate that design/information and not follow the old 2021 recommendation. Thanks for digging, though!

**Comment by jilladams** [comment]
@FranECross this one is ready to verify.

**Comment by chriskim2311** [comment]
@laflannery DST has a PR out for this fix https://github.com/department-of-veterans-affairs/vets-website/pull/33185

**Comment by FranECross** [comment]
@chriskim2311 I'm going to continue testing this today (there are SO many scenarios) but everything checks out so far, so I'm closing as complete and will create a new ticket if necessary for anything that may come up (which I don't foresee). cc @jilladams  I also added this to the demo doc.
